### PR TITLE
Added minimal config file for iam-components helm-chart

### PR DESCRIPTION
**Pull request by beknazaresenbek**

### CODE REVIEW

File: config/iam-components.yaml [hunk 1]
```diff
@@ -0,0 +1,1152 @@
...
+
+vcwaltid:
+  ingress:
+    enabled: false
+
+  fullnameOverride: waltid-iam
+  deployment:
+    image:
+      repository: quay.io/fiware/waltid
+      tag: 1.14.1-SNAPSHOT
```
Comment (dwendland) [inline config/iam-components.yaml:44]: This applies also to the other components:
Shouldn't the IAM chart already set the correct image tags and registries to be used? In this case those parameters can be removed.

File: config/iam-components.yaml [hunk 2]
```diff
@@ -0,0 +1,1152 @@
+postgresql:
```
Comment (dwendland) [inline config/iam-components.yaml:1]: In general about the whole values file:
Can this be reduced to a minimal example? I suggest to leave out parameters that are set by the chart default values already (like image tags, etc), and rather concentrate on the parameters that are important to be configured (and maybe adding a comment for them what they are used for). 
Compare to the example for the access node: https://github.com/DOME-Marketplace/integration-guide/blob/iam-minimal-config/config/accessnode.yaml

**Review by dwendland — CHANGES_REQUESTED** [review]

File: config/iam-components.yaml (hunk 1 above)
Comment (beknazaresenbek) [reply config/iam-components.yaml:44]: yes, you're right, I'll update it

File: config/iam-components.yaml (hunk 2 above)
Comment (beknazaresenbek) [reply config/iam-components.yaml:1]: I'll update it too, thanks

**Review by dwendland — APPROVED** [review]

**Pull request merged**